src/Configuration/Parser/BaseParser.php

```diff
@@ -54,5 +69,14 @@ public function getFilenames(): array
         return $this->filenames;
     }
 
+    public function getFilename()
+    {
+        return $this->filename;
+    }
+
+    public function getFilenameLocalOverrides() {
+        return preg_replace('/([a-z0-9_-]+).(ya?ml)$/i', '$1_local.$2', $this->filename);
```

Comment:
why you don't do this Symfony way? https://symfony.com/doc/current/configuration/configuration_organization.html

Reply:
Like the previous time you asked me this: I don't like that component.. It's too cumbersome for my tastes. ;-)

Reply:
But people are used to use it in Symfony projects. I'd use what community loves.

Reply:
Nah, I'd really like to keep this as is. :-)

Reply:
How can I have more than "normal" and "local" environments then?

Reply:
That can be a feature-request for later. ;-)

Reply:
Later it can be hard to refactor

Reply:
Doing too much now will add needless complexity.